Doc/library/warnings.rst

```diff
@@ -199,15 +199,17 @@ precedence over earlier ones).
 
 Commonly used warning filters apply to either all warnings, warnings in a
 particular category, or warnings raised by particular modules or packages.
-Some examples::
+Some examples:
+
+.. code-block:: none
 
    default                      # Show all warnings (even those ignored by default)
    ignore                       # Ignore all warnings
    error                        # Convert all warnings to errors
    error::ResourceWarning       # Treat ResourceWarning messages as errors
    default::DeprecationWarning  # Show DeprecationWarning messages
    ignore,default:::mymodule    # Only report warnings triggered by "mymodule"
-   error:::mymodule[.*]         # Convert warnings to errors in "mymodule"
+   error:::mymodule(\..*)?      # Convert warnings to errors in "mymodule"
                                 # and any subpackages of "mymodule"
```

Comment:
While at it and to be precise, the regex actually matches any _submodules_, not just subpackages.